### ISSUE TITLE
Confusing statement 

### ISSUE THREAD

**Issue by dcpopescu** [MicrosoftDocs/windows-itpro-docs#5244]
Hi,
Are this benefits included in W10 E3 or not?

These benefits include planning services, Microsoft Desktop Optimization (MDOP), Windows Virtual Desktop Access Rights, Windows-To-Go Rights, Windows Roaming Use Rights, Windows Thin PC, Windows RT Companion VDA Rights, and other benefits.

Thank you

**Comment by mypil** [comment]
@dcpopescu - May we know which Microsoft document link you used in reference to this issue?
Please share it with us so we can investigate further. Thank you.

**Comment by dcpopescu** [comment]
Hi Joyce,

This is the article:

https://docs.microsoft.com/en-us/windows/deployment/windows-10-enterprise-e3-overview


This part of the document is unclear:


How does the Windows 10 Enterprise E3 in CSP program compare with Microsoft Volume Licensing Agreements and Software Assurance?

  *   Microsoft Volume Licensing<https://www.microsoft.com/licensing/default.aspx> programs are broader in scope, providing organizations with access to licensing for all Microsoft products.
  *   Software Assurance<https://www.microsoft.com/Licensing/licensing-programs/software-assurance-default.aspx> provides organizations with the following categories of benefits:
     *   Deployment and management. These benefits include planning services, Microsoft Desktop Optimization (MDOP), Windows Virtual Desktop Access Rights, Windows-To-Go Rights, Windows Roaming Use Rights, Windows Thin PC, Windows RT Companion VDA Rights, and other benefits.
     *   Training. These benefits include training vouchers, online e-learning, and a home use program.
     *   Support. These benefits include 24x7 problem resolution support, backup capabilities for disaster recovery, System Center Global Service Monitor, and a passive secondary instance of SQL Server.
     *   Specialized. These benefits include step-up licensing availability (which enables you to migrate software from an earlier edition to a higher-level edition) and to spread license and Software Assurance payments across three equal, annual sums.

In addition, in Windows 10 Enterprise E3 in CSP, a partner can manage your licenses for you. With Software Assurance, you, the customer, manage your own licenses.

In summary, the Windows 10 Enterprise E3 in CSP program is an upgrade offering that provides small- and medium-sized organizations easier, more flexible access to the benefits of Windows 10 Enterprise edition, whereas Microsoft Volume Licensing programs and Software Assurance are broader in scope and provide benefits beyond access to Windows 10 Enterprise edition.

Windows 10 Enterprise E3 doesn’t include MDOP?

Thank you,
________________________________

**Comment by MaratMussabekov** [comment]
Hello @dcpopescu,
unfortunately no, Windows E3 in CSP doesn't provide listed benefits. It only allows users who have Windows 10 Pro licenses to get Windows 10 Enterprise features via subscription (benefits listed by you are not Windows 10 Enterprise features, they are Software Assurance features).
Thank you

**Comment by mypil** [comment]
@dcpopescu - From our understanding, the issue you raised has been answered by MaratMussabekov. If you feel it hasn't been answered, please re-open this issue.

Thank you for your contribution to make the docs better! Much appreciated!
